### PR TITLE
[mainnet] Deploy the newly updated Smart Escrow Contract

### PR DESCRIPTION
**Pull request by stevieraykatz**
The escrow contract was updated with this PR: https://github.com/base-org/contracts/pull/81
And audited in this PR: https://github.com/base-org/contracts/pull/85

### CODE REVIEW

File: mainnet/2024-06-05-reredeploy-smart-escrow/.env [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+OP_TOKEN=0x4200000000000000000000000000000000000042
+
+BENEFACTOR=0x2501c477d0a35545a387aa4a3eee4292a9a8b3f0
+BENEFICIARY=0x635Fb974F09B269Bc750bF96338c29cF41430125
+BENEFACTOR_OWNER=0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0
+BENEFICIARY_OWNER=0x6e1DFd5C1E22A4677663A81D24C6BA03561ef0f6
+NESTED_SAFE=0x0a7361e734cf3f0394B0FC4a45C74E7a4Ec70940
+START=1720674000
+CLIFF_START=1724976000
+END=1882764000
```
Comment (stevieraykatz) [inline mainnet/2024-06-05-reredeploy-smart-escrow/.env:16]: Needs to be July 11, 2029 : 1878462000

Comment (stevieraykatz) [reply mainnet/2024-06-05-reredeploy-smart-escrow/.env]: Updated to reflect updated END time.

File: mainnet/2024-06-05-reredeploy-smart-escrow/script/DeploySmartEscrow.s.sol [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        );
+        require(smartEscrow.start() == START, "DeploySmartEscrow: start time not set correctly");
+        require(smartEscrow.cliffStart() == CLIFF_START, "DeploySmartEscrow: cliff start time not set correctly");
+        require(smartEscrow.end() == END, "DeploySmartEscrow: end time not set correctly");
+        require(smartEscrow.vestingPeriod() == VESTING_PERIOD_SECONDS, "DeploySmartEscrow: vesting period incorrect");
+        require(smartEscrow.initialTokens() == INITIAL_TOKENS, "DeploySmartEscrow: number of initial tokens incorrect");
+        require(smartEscrow.vestingEventTokens() == VESTING_EVENT_TOKENS, "DeploySmartEscrow: number of vesting event tokens incorrect");
+        require(smartEscrow.benefactor() == BENEFACTOR, "DeploySmartEscrow: benefactor incorrect"); 
+        require(smartEscrow.beneficiary() == BENEFICIARY, "DeploySmartEscrow: beneficiary incorrect");
+        require(smartEscrow.released() == 0, "DeploySmartEscrow: initial released value must bex zero");
```
Comment (anikaraghu) [inline mainnet/2024-06-05-reredeploy-smart-escrow/script/DeploySmartEscrow.s.sol:43]: nit: typo
```suggestion
        require(smartEscrow.released() == 0, "DeploySmartEscrow: initial released value must be zero");
```

**Review by anikaraghu — APPROVED** [review]
LGTM thanks! checked agains [that reference](https://docs.google.com/document/d/1_0nXEqEKoetmQmWRC-yZOf1aAwmsbYqFMPw_RMLAz4Y/edit) you had put together for reviewers - btw can you also add the cliffStart to it?

File: mainnet/2024-06-05-reredeploy-smart-escrow/.env [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+OP_TOKEN=0x4200000000000000000000000000000000000042
+
+BENEFACTOR=0x2501c477d0a35545a387aa4a3eee4292a9a8b3f0
+BENEFICIARY=0x635Fb974F09B269Bc750bF96338c29cF41430125
+BENEFACTOR_OWNER=0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0
+BENEFICIARY_OWNER=0x6e1DFd5C1E22A4677663A81D24C6BA03561ef0f6
+NESTED_SAFE=0x0a7361e734cf3f0394B0FC4a45C74E7a4Ec70940
+START=1720674000
+CLIFF_START=1724976000
```
Comment (anikaraghu) [inline mainnet/2024-06-05-reredeploy-smart-escrow/.env:15]: this is `Fri Aug 30 2024 00:00:00 GMT+0000` - is that correct?

Comment (stevieraykatz) [reply mainnet/2024-06-05-reredeploy-smart-escrow/.env:15]: Confirmed by Ben, based on date of first token received at Nested Multisig: https://optimistic.etherscan.io/tx/0x6d771624ecbff8704edfc5cbf3bff57b8bac9584775457b44e9872081cd0a733

**Review by anikaraghu — APPROVED** [review]

**Comment by cb-heimdall** [comment]
<h1>✅ Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
✅
<code>1/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Comment by stevieraykatz** [comment]
:fire:

**Pull request merged**